mix.exs

```diff
@@ -19,14 +19,15 @@ defmodule Barlix.Mixfile do
   end
 
   def application do
-    [applications: [:logger, :png]]
+    [applications: [:logger, :png, :eex]]
   end
 
   defp deps do
     [
-      {:png, "~> 0.1"},
+      {:png, "~> 0.2"},
+      {:dialyxir, "~> 1.0", only: [:dev], runtime: false},
       {:ex_doc, "~> 0.20.2", only: :dev},
-      {:mix_test_watch, "~> 0.2", only: :dev},
+      # {:mix_test_watch, "~> 0.2", only: :dev},
```

Comment:
I don't use this much as well, this can be just removed

Reply:
Ok, I removed it. Actually this had issues to compile and I didn't want to bother to look into it.